composer.json

```diff
@@ -213,7 +213,6 @@
                 "Allow created date to be visible in the form. @see https://github.com/ec-europa/rdf_entity/pull/10": "https://patch-diff.githubusercontent.com/raw/ec-europa/rdf_entity/pull/10.patch"
             },
             "drush/drush": {
-                "Allow updating modules with new service dependencies @see https://github.com/drush-ops/drush/pull/3206": "https://patch-diff.githubusercontent.com/raw/drush-ops/drush/pull/3206.patch",
```

Comment:
It's bizarre that this change to the `composer.json` file without a corresponding update to `composer.lock` causes the patch to be ignored. Normally the `composer.lock` should be the canonical source of information.

It appears this is a known issue: https://github.com/cweagans/composer-patches/issues/12